src/chapters/databases-part2/db-setup.rst

```diff
@@ -0,0 +1,101 @@
+Database Setup
+==============
+
+The Minesweeper game will use a combination of session cookies and database
+operations to store important information. Let's begin by getting the database
+file ready.
+
+Start With an Outline
+---------------------
+
+Before we write any code, we need to think about what data the game needs in
+order to work. In particular, we want to identify the information to save in
+the game's database. There are LOTS of possibilities! However, for our basic
+game, we want to start small. We can always refine the database structure
+later!
+
+Consider the data that needs to *persist*. For example, if a user leaves the
+game in the middle of play and comes back later, what needs to go into the
+database to keep a record of their progress?
+
+For starters, our database should save:
+
+#. Information about each mine hidden on the board. This includes a primary key
+   value as well as a location (row letter plus column number).
+#. Information about each individual cell on the board. This includes a primary
+   key, the cell's location, the number of mines in surrounding cells, whether
+   the user has clicked the cell, and whether the cell contains a mine.
+
+This outline suggests at 2 tables in the database - one for a list of mines,
```

Comment:
Weird wording - This outline suggests at 2 tables

Reply:
Fixed!